modules/core/src/shaderlib/shadow/shadow.ts

```diff
@@ -190,7 +190,9 @@ function createShadowUniforms(
   if (!shadowEnabled || !opts.shadowMatrices || !opts.shadowMatrices.length) {
     return {
       shadow_uDrawShadowMap: false,
-      shadow_uUseShadowMap: false
+      shadow_uUseShadowMap: false,
+      shadow_uShadowMap0: opts.dummyShadowMap,
```

Comment:
Why is this required?

Reply:
v9 throws if a sampler is left unbound, even if it is not used.